common/lumens.js

```diff
@@ -1,10 +1,10 @@
 // This file contains functions used both in backend and frontend code.
 // Will be helpful to build distribution stats API.
-import axios from "axios";
```

Comment:
needed to remove babel on the backend because of a [bug with sequelize](https://github.com/sequelize/sequelize/issues/7840) (annoyingly since sequelize will be removed soon). So changed `common/lumens.js` to use commonJS `common/require` instead of `import/export` which works for both the frontend and backend